src/shared/components/oncoprint/SortUtils.ts

```diff
@@ -302,6 +303,42 @@ export function getClinicalTrackSortComparator(track: ClinicalTrackSpec) {
     };
 }
 
+export function getClinicalTrackInitialSortDirection(
+    track: ClinicalTrackSpec
+): 0 | 1 | -1 {
+    let initialSortOrder: 0 | 1 | -1 = 0;
+
+    const defaultTracks = ServerConfigHelpers.parseDefaultOncoprintClinicalTracks(
+        getServerConfig().oncoprint_clinical_tracks_show_by_default!
+    );
+
+    defaultTracks.forEach(t => {
```

Comment:
little bit silly to iterate through whole thing.  just find track element using defaultTracks.find()

Reply:
This functionality has been removed: SortUtils doesn't need to find default tracks anymore